### PR TITLE
Code standardization and refactoring

### LINKED ISSUE #4: Code standardization and refactoring

### PR DESCRIPTION
**Pull request by natamo**
fixed #4

### CODE REVIEW

**Review by siddhartha2202 — APPROVED** [review]
Perfect

File: demo/features/calculator.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+class Calculator:
+
+    def __init__(self):
+        self.__set_params(0, 0)
+
+
+    def __set_params(self, num_arg1, num_arg2):
+        self.num_arg1 = num_arg1 
+        self.num_arg2 = num_arg2
+
```
Comment (Kid-666) [inline demo/features/calculator.py:21]: Does not meet the standard

File: test/features/calculator_test.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+def test_rest():
+    calculator = Calculator()
+    assert calculator.rest(4, 2) == 2
+    
+def test_mult():                                               
+    calculator = Calculator()                                 
+    assert calculator.mult(2, 2) == 4
+
```
Comment (Kid-666) [inline test/features/calculator_test.py]: Does not meet the standard

**Review by Kid-666 — CHANGES_REQUESTED** [review]
Please apply recommendations

File: demo/features/calculator.py (hunk 1 above)
Comment (natamo) [reply demo/features/calculator.py:21]: Fixed

File: test/features/calculator_test.py (hunk 2 above)
Comment (natamo) [reply test/features/calculator_test.py]: Fixed

**Review by Kid-666 — APPROVED** [review]

**Pull request merged**